src/Helmet.js

```diff
@@ -270,7 +270,11 @@ const Helmet = (Component) => {
         shouldComponentUpdate(nextProps) {
             return !deepEqual(this.props, nextProps);
         }
-
+        
+        // Component.peak comes from react-side-effect:
+        // For testing, you may use a static peek() method available on the returned component.
+        // It lets you get the current state without resetting the mounted instance stack.
+        // Don’t use it for anything other than testing.
```

Comment:
I could see `peek` being used outside of testing.  A user may want to simply view the current state in Helmet on the client side in an on-demand fashion.  Why would we want to discourage people from using it?